frontend/src/old-pages/Configure/HeadNode.tsx

```diff
@@ -294,6 +309,13 @@ function HeadNode() {
             If enabled, restrict access to IMDS (and thus instance credentials) to users with superuser permissions. For more information, see <a rel="noreferrer" target="_blank" href='https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html#instance-metadata-v2-how-it-works'>How instance metadata service version 2 works</a> in the <i>Amazon EC2 User Guide for Linux Instances</i>.
           </HelpTooltip>
         </div>
+        <div key="memory-based-scheduling-enabled" style={{display: "flex", flexDirection: "row", alignItems: "center", justifyContent: "space-between"}}>
+          <Toggle
+            checked={memoryBasedSchedulingEnabled || false} onChange={toggleMemoryBasedSchedulingEnabled}>Slurm Memory Based Scheduling Enabled</Toggle>
+          <HelpTooltip>
+            If enabled, users may use <i>--mem</i> to specify the amount of memory per node required by a job. For more information, see <a rel="noreferrer" target="_blank" href='https://slurm.schedmd.com/cgroup.conf.html#OPT_ConstrainRAMSpace'>the Slurm documentation for ConstrainRAMSpace</a>
+          </HelpTooltip>
+        </div>
```

Comment:
issue(blocking): For all the new code, we want to adopt `i18n` instead of hardcoding strings. You can find an example here https://github.com/aws-samples/pcluster-manager/pull/175/commits/fdc6b77987c87a26f51dbc8da5d371d95ef80601

For text styling (`<i>`) and links `<a />` it's probably going to be a bit tricky, but we can find a solution